pkg/api/serialization_test.go

```diff
@@ -532,6 +532,10 @@ func originFuzzer(t *testing.T, seed int64) *fuzz.Fuzzer {
 				securityapi.FSTypeRBD,
 				securityapi.FSTypeSecret}
 			scc.Volumes = []securityapi.FSType{volumeTypes[c.Rand.Intn(len(volumeTypes))]}
+
+			// match to the defaulting logic that is used to be backward compatible
+			scc.AllowPrivilegeEscalation = true
```

Comment:
This looks off.  Why doesn't `false` roundtrip?

Reply:
I suspect that to the original object were applied defaults that always set this value to `true`? But I'm not sure. I just know that it works :(